tests/nn/conftest.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+import pytest
+import torch
+
+from pathpyG.core.IndexMap import IndexMap
+from pathpyG.core.PathData import PathData
+
+
+@pytest.fixture
+def simple_paths() -> PathData:
```

Comment:
This is a copy of `simple_paths()` in `tests/core/conftest.py`, isn't it? I recommend creating one `conftest.py` at a hierarchy level above i.e. `tests/conftest.py`. You can then access it in both submodules `nn` and `core`.

Reply:
Right now, you are right. But this is only a temporary function that I copied, i..e I expect the test cases to differ eventuallt, so I propose to keep it as it is.